docs/Components.md

```diff
@@ -955,50 +955,50 @@ Read more about [peripherals](Supported-Peripherals.md).
-| 7766 | Switch_d23     |                                                                                                         |
-| 7767 | Switch_d24     |                                                                                                         |
-| 7768 | Switch_d25     |                                                                                                         |
-| 7769 | Switch_d26     |                                                                                                         |
-| 7770 | Switch_d27     |                                                                                                         |
-| 7771 | Switch_d28     |                                                                                                         |
+| 7680 | Button_d1      | Button with internal pull-down resistor , (Active High)                                                 |
+| 7681 | Button_d2      | Button with internal pull-down resistor , (Active High)                                                 |
+| 7682 | Button_d3      | Button with internal pull-down resistor , (Active High)                                                 |
+| 7683 | Button_d4      | Button with internal pull-down resistor , (Active High)                                                 |
+| 7684 | Button_d5      | Button with internal pull-down resistor , (Active High)                                                 |
+| 7685 | Button_d6      | Button with internal pull-down resistor , (Active High)                                                 |
+| 7686 | Button_d7      | Button with internal pull-down resistor , (Active High)                                                 |
+| 7687 | Button_d8      | Button with internal pull-down resistor , (Active High)                                                 |
+| 7712 | Button_id1     | Button with internal pull-down resistor , Inverted (Active Low)                                         |
+| 7713 | Button_id2     | Button with internal pull-down resistor , Inverted (Active Low)                                         |
+| 7714 | Button_id3     | Button with internal pull-down resistor , Inverted (Active Low)                                         |
+| 7715 | Button_id4     | Button with internal pull-down resistor , Inverted (Active Low)                                         |
+| 7716 | Button_id5     | Button with internal pull-down resistor , Inverted (Active Low)                                         |
+| 7717 | Button_id6     | Button with internal pull-down resistor , Inverted (Active Low)                                         |
+| 7718 | Button_id7     | Button with internal pull-down resistor , Inverted (Active Low)                                         |
+| 7719 | Button_id8     | Button with internal pull-down resistor , Inverted (Active Low)                                         |
```

Comment:
I believe you swapped Active High and active Low
Button_1, Button_n1  and Button_d1 are all active low
Button_i1, Button_in1 and Button_id1 are all active high